vendor/github.com/openshift/cluster-version-operator/lib/resourcemerge/core.go

```diff
@@ -56,6 +56,15 @@ func ensurePodSpec(modified *bool, existing *corev1.PodSpec, required corev1.Pod
 		}
 		ensureContainer(modified, existingCurr, required)
 	}
+	var requiredContainers []corev1.Container
```

Comment:
How is this change related to the unit tests?

Reply:
https://github.com/openshift/cluster-version-operator/pull/178